src/Features.php

```diff
@@ -101,6 +103,15 @@ private static function get_fields(): array {
 				'id'   => self::BETA_BLOCKS,
 				'type' => 'checkbox',
 			],
+			[
```

Comment:
Included it here for now, we can probably come up with a better menu location.

Reply:
Let's create a new category for this (Planet 4 > Notifications), so we have Features only for features flags.